packages/quantic/force-app/main/default/lwc/descriptionStrengthIndicator/descriptionStrengthIndicator.js

```diff
@@ -0,0 +1,66 @@
+import { LightningElement, api } from 'lwc';
+
+/**
+ * The `descriptionStrengthIndicator` component is a dynamic indicator that shows the user if their case description has enough details so far. It also give hints to the user as to what information they show include in the description. 
+ * @example
+ * <c-description-strength-indicator></c-description-strength-indicator>
+ */
+export default class DescriptionStrengthIndicator extends LightningElement {
+    /**
+     * The message to be shown to the user.
+     * @api
+     * @type {string}
+     * @defaultValue 'Don’t know what to write?'
+     */
+    @api
+    helpLabel = "Don’t know what to write?";
```

Comment:
strings are delimited with `'` in the project so it would be better if we keep that consistent.